.gitignore

```diff
@@ -1,4 +1,3 @@
 target
 .ccls-cache
-Cargo.lock
```

Comment:
I would like to avoid committing the lock file. I think it is still best to ensure rquickjs is tested against the latest dependencies if possible. The linked blog post says that they do not have a default recommendation not that they recommend against not committing the lockfile.